main.cpp

```diff
@@ -133,30 +199,44 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 int main()
 {
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
-    auto carRented = rentACar(6, 2); 
-    
-    //1)
-    
-    //2)
-    
-    //3)
+    // auto carRented = rentACar(6, 2); 
     
+    //1)
+    int signalArray[] = {1, 2, 3};
+    sendSignal(signalArray, 3);
+    //2)
+    char dog[] = "dog";
+    auto log = changeFirstLetter(dog, 'l');
+    //3)
+    int toppings[] = {1, 2, 3};
+    auto orderID = placePizzaOrder(1, toppings);
     //4)
-    
+    auto orderStatus = getOrderStatus(orderID);
     //5)
-    
+    playSound(1, 1.1f);
     //6)
-    
+    launchProjectile(40, 0);
     //7)
-    
-    //8)
-    
+    run(100);
+    //8)
+    char text[] = "Hello World.";
+    auto note = createNote(text);

```

Comment:
no arrays yet (especially not raw C-style arrays. this is C++, remember?)
they haven't been covered, nor have the gotchas been explained for them.
- [ ] revise